vizro-ai/hatch.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
   "mkdocs-material",
   "mkdocs-git-revision-date-localized-plugin",
   "mkdocstrings[python]",
+  "mkdocstrings-python<1.10.4",
```

Comment:
Are "mkdocstrings[python]" and "mkdocstrings-python" different? If not combine?

Reply:
No, they're different: `mkdocstrings[python]` has `mkdocstrings-python` as an unpinned dependency.